### DISCUSSION TITLE
How to make results of code cell display above code annotations?

### DISCUSSION THREAD

**Discussion by alecloudenback** [quarto-dev/quarto-cli#8279] [Q&A]
### Description

[Code annotations](https://quarto.org/docs/authoring/code-annotation.html ) seem to display executed cell results *below* the annotations. Is there a configuration that would have the result show above the annotations? With a longer set of annotations it can be weird for the result of the code to be so far removed from the code cell.

Example:

    ```{julia}
    a = 1 #<1>
    a+5 #<2>
    ```
    1. This is an annotation.
    2. A second annotation. The result of this is `6`.

Renders as the following, with the code result (`6`) appearing below the annotations:

<img width="641" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/711879/631c8de2-6be2-4428-a2d1-d2ba85cd27cf">

With the question being: can the result (`6`) be displayed above the numbered list in the rendered output?

**Comment by mcanouil** [comment]
I am not sure to understand.
Would you mind sharing a reproducible example?

**Reply by alecloudenback** [reply]
I updated the OP with an example.

**Reply by cscheid** [reply]
That's not possible right now - the annotations are considered part of the code, and the result is separate from that.
